app/models/reports/schema_v2.rb

```diff
@@ -0,0 +1,7 @@
+module Reports
```

Comment:
I'm not sure I understand this. Is it intended to be a mixin?

Reply:
removing for now